.gitignore

```diff
@@ -22,7 +22,8 @@
 /user.bazelrc
 
 # Compilation database used by clangd
-compile_commands.json
+/compile_commands.json
+/external
```

Comment:
Removing /external because it's tool-dependent, but I feel weird about removing compile_commands.json since it's pre-existing (and may come from other tools, without ever running the hedronvision approach)